ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -96,6 +101,19 @@ public class BottomSheetController: UIViewController {
         overflowView.backgroundColor = Colors.NavigationBar.background
         view.addSubview(overflowView)
 
+        if shouldShowDimmingView {
+            view.addSubview(dimmingView)
+            NSLayoutConstraint.activate([
+                dimmingView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+                dimmingView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+                dimmingView.topAnchor.constraint(equalTo: view.topAnchor),
+                dimmingView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+            ])
+        }
+
+        view.bringSubviewToFront(bottomSheetView)
+        view.bringSubviewToFront(overflowView)
```

Comment:
can we switch when we add the overflowView as subview so we don't call it in line 102 and then again in 115?